package.json

```diff
@@ -11,7 +12,10 @@
         "date-fns": "^2.28.0",
         "formik": "^2.2.9",
         "ky": "^0.31.0",
+        "moment": "^2.29.4",
```

Comment:
`date-fns` has already been added as a dependency, if possible try using it instead of Moment as it's more performant and is smaller.